src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -927,6 +910,94 @@ catch (const SettingsTypedDeserializationException& e)
     return *settings;
 }
 
+void CascadiaSettings::_researchOnLoad()
+{
+    // Only do this if we're actually being sampled
+    if (TraceLoggingProviderEnabled(g_hSettingsModelProvider, 0, MICROSOFT_KEYWORD_MEASURES))
+    {
+        // GH#13936: We're interested in how many users opt out of useAtlasEngine,
+        // indicating major issues that would require us to disable it by default again.
+        {
+            size_t enabled[2]{};
+            for (const auto& profile : _activeProfiles)
+            {
+                enabled[profile.UseAtlasEngine()]++;
+            }
+
+            TraceLoggingWrite(
+                g_hSettingsModelProvider,
+                "AtlasEngine_Usage",
+                TraceLoggingDescription("Event emitted upon settings load, containing the number of profiles opted-in/out of useAtlasEngine"),
+                TraceLoggingUIntPtr(enabled[0], "UseAtlasEngineDisabled", "Number of profiles for which AtlasEngine is disabled"),
+                TraceLoggingUIntPtr(enabled[1], "UseAtlasEngineEnabled", "Number of profiles for which AtlasEngine is enabled"),
+                TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES),
+                TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage));
+        }
+
+        // ----------------------------- RE: Themes ----------------------------
+        const auto numThemes = GlobalSettings().Themes().Size();
+        const auto themeInUse = GlobalSettings().CurrentTheme().Name();
+
+        // system: 0
+        // light: 1
+        // dark: 2
+        // a custom theme: 3
+        const auto themeChoice = themeInUse == L"system" ? 0 :

```

Comment:
By and large, our data will show that users are using the theme from defaults.json.

Should we only log this if the user has overridden the default theme? `GlobalAppSettings` supports all the Inheritable stuff, so we should be able to detect that.

Reply:
(I'm guessing it's like, 99.9% "whatever we put in the defaults")